site-docs/src/main/paradox/entity/usage/index.md

```diff
@@ -56,6 +56,10 @@ The client exposes 12 methods
       entityType: EntityType,
       identifier: Identifier
   ): F[String]
+
+  def getPreservicaNamespaceVersion(
```

Comment:
There's a line at the top that says that this exposes 12 methods but it's 15 now. Would you mind updating it.